pandas/sparse/sparse.pyx

```diff
@@ -71,7 +71,7 @@ cdef class IntIndex(SparseIndex):
     def check_integrity(self):
         """
         Only need be strictly ascending and nothing less than 0 or greater than
-        totall ength
+        total length
         """
         pass
```

Comment:
@jreback : Is there any reason that this is implemented (not required from super-class)?  I don't see it used anywhere by `IntIndex`, and the implementation is just `pass` even though the documentation says it should do something.

If it is supposed to be used, should we move this method to the `SparseIndex` class (common method among the two sub-classes in that file)?